src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
Perhaps we can extend notifySubcribers to take a third param? I'd like to avoid wrapping the value because you can still directly publish on the all event. That would let us pass the original event along and reuse notifySubcribers, but it would change the arity of the function. That could cause issues - for example if people used it as the callback in an _.each to notify events from an object